x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -179,23 +179,30 @@ private static Object valueMapper(CsvTestUtils.Type type, Object value) {
         if (value == null) {
             return "null";
         }
-        if (type == CsvTestUtils.Type.GEO_POINT) {
-            // GeoPoint tests are failing in clustered integration tests because of tiny precision differences at very small scales
+        if (type == CsvTestUtils.Type.GEO_POINT || type == CsvTestUtils.Type.CARTESIAN_POINT) {
+            // Point tests are failing in clustered integration tests because of tiny precision differences at very small scales
             if (value instanceof GeoPoint point) {
```

Comment:
Can we really get a GeoPoint here? I would expect only WKT can get to this point?

Reply:
You are right, older code would have reached this, but I don't think the current code ever does. I could clean this up in another PR.